libs/utils/utils.js

```diff
@@ -412,6 +413,20 @@ export function decorateSVG(a) {
   }
 }
 
+export function decorateVideo(a) {
```

Comment:
Why can't this be an autoblock? Seems like that's basically what we're doing already.

The great part about autoblocks / link blocks is that they can run inside other blocks... marquee > inline-video.

Reply:
How would you use regex (`media_.*.mp4$`) in auto block though? https://github.com/adobecom/milo/blob/a0b6e00dc8e9c773665f4d4743b77af53500af4c/libs/utils/utils.js#L418

Reply:
I agree with @auniverseaway, having this as an autoblock would be great. How about supporting regular expressions for autoblocks by extending the `decorateAutoBlocks` method?
https://github.com/adobecom/milo/blob/main/libs/utils/utils.js#L419